db/migrate/20231201083922_add_provider_to_establishments.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddProviderToEstablishments < ActiveRecord::Migration[7.1]
+  def change
+    change_table :establishments, bulk: true do |t|
+      t.string :students_provider
```

Comment:
not null à priori non ?

Reply:
J'ai pas vraiment de valeur par défaut à mettre, donc la migration va planter si je met juste `t.string :students_provider, null: false`.

Je préfère avoir `nil` qu'une valeur genre `no_provider` par défaut.